test/cudnn.cu

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <cuda_runtime.h>
+#include <cudnn.h>
+
+/**
+ * Minimal example to apply sigmoid activation on a tensor 
+ * using cuDNN with cudaMalloc instead of cudaMallocManaged.
+ **/
+int main(int argc, char** argv)
+{    
+    int numGPUs;
+    cudaGetDeviceCount(&numGPUs);
+    std::cout << "Found " << numGPUs << " GPUs." << std::endl;
+    cudaSetDevice(0); // use GPU0
+    int device;
+    struct cudaDeviceProp devProp;
+    cudaGetDevice(&device);
+    cudaGetDeviceProperties(&devProp, device);
+    std::cout << "Compute capability:" << devProp.major << "." << devProp.minor << std::endl;
+
+    cudnnHandle_t handle_;
+    cudnnCreate(&handle_);
+    std::cout << "Created cuDNN handle" << std::endl;
+
+    // create the tensor descriptor
+    cudnnDataType_t dtype = CUDNN_DATA_FLOAT;
+    cudnnTensorFormat_t format = CUDNN_TENSOR_NCHW;
+
+    int n = 1, c = 1, h = 1, w = 10;
+    int NUM_ELEMENTS = n * c * h * w;
+    cudnnTensorDescriptor_t x_desc;
+
+    cudnnCreateTensorDescriptor(&x_desc);
+    cudnnSetTensor4dDescriptor(x_desc, format, dtype, n, c, h, w);
+
+    // create and initialize the tensor
+    float *h_x = new float[NUM_ELEMENTS];
+    for (int i = 0; i < NUM_ELEMENTS; i++) h_x[i] = i * 1.00f;
+    
+    float *d_x; // device mem
+    cudaMalloc(&d_x, NUM_ELEMENTS * sizeof(float));
```

Comment:
@kevmo314 I had to change this file to use cudaMalloc instead of cudaMallocManaged until we figure out the latter.